AncientReligions/common/on_actions/00_on_actions.txt

```diff
@@ -0,0 +1,1835 @@
+# ACR impacted on_actions: vanilla events need to be kept, as it does a full override of events & random_events lists
+# It's easier to keep whole file for merge: every patch will require a merge anyway even on the portion modified for ACR...
```

Comment:
I've put back the other on_action stuff from vanilla as my diftool was going crazy. I've discovered since then that difftool "skew tolerance" can be increased to make a smarter diff.
Yet, an ACR_on_actions.txt loaded after vanilla could mislead in thinking no merge is required with 00_on_actions.txt, while it's the contrary.